CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v0.17.3
+- Move SpeedyParameters to SpeedyWeatherInternals
+
```

Comment:
You want #882 to be merged first? If then I'd still put this under unreleased and then we can have a separate PR just to bump to v0.17.3 where then introduce `## v0.17.3`

Reply:
That would cause precompile errors in the current release of SpeedyWeather.

Reply:
#882 is merged now, so I think you can also just go ahead an bump the version of both after this is merged.

Reply:
Given @bgroenks96 already changed the SpeedyWeather version in Project.toml we apprently do this in one and not a separate PR just to bump the version. Happy with this